### PR TITLE
Daypass fixes

### PR DESCRIPTION
**Pull request by adi10197**
The email of hostelsuperintendent while applying for daypass for production needs to be checked. @SebastinSanty  Kindly review and suggest fixes.
Issue #134

### CODE REVIEW

File: swd/main/models.py [hunk 1]
```diff
@@ -81,6 +81,15 @@ class Warden(models.Model):
     def __str__(self):
         return self.hostel + ' ' + self.name + ' ' + self.email + ' ' + self.chamber
 
+class Hostel_Super_Intendent(models.Model):
```
Comment (SebastinSanty) [inline swd/main/models.py]: I don't think Super_Intendent are two separate words

File: swd/main/models.py [hunk 2]
```diff
@@ -81,6 +81,15 @@ class Warden(models.Model):
...
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    name = models.CharField(max_length=50, null=True, blank=True)
+    email = models.EmailField(null=True, blank=True)
+    hostel = models.CharField(max_length=5, choices=HOSTELS, null=True, blank=True)
```
Comment (SebastinSanty) [inline swd/main/models.py]: One superintendent handles multiple hostels. How do you think we can handle that. With this model, we need to create a new object for each hostel. I am OK with that. Is there any other way, you think we can handle this?

**Comment by SebastinSanty** [comment]
Any updates? @adi10197

File: swd/main/models.py (hunk 2 above)
Comment (adi10197) [reply swd/main/models.py]: If we make one object handle multiple hostels, it becomes superintendent specific. But the daypass form application is hostel specific. Makes it easier with the code already present. Right now, say Venu is both the superintendent of AH4, and AH5, we'll have to create two different users, ah4superintendent, and ah5superintendent, these two being what he'll have to use to view AH4, and AH5's daypass requests seperately. After creating a user, a superintendent's instance has a name field which can contain the same name for both objects.

**Comment by SebastinSanty** [comment]
There are some other issues to be fixed a bit. I'll keep it for another PR.

**Pull request merged**